RustFxA/FirefoxAccountSignInViewController.swift

```diff
@@ -95,11 +95,14 @@ class FirefoxAccountSignInViewController: UIViewController, Themeable {
         }
     }
 
-    private lazy var scanButton: ResizableButton = .build { button in
+    private lazy var scanButton: PrimaryRoundedButton = .build { button in
+        let viewModel = PrimaryRoundedButtonViewModel(
+            title: .FxASignin_QRScanSignin,
+            a11yIdentifier: AccessibilityIdentifiers.Settings.FirefoxAccount.qrButton
+        )
+        button.configure(viewModel: viewModel)
         button.layer.cornerRadius = UX.buttonCornerRadius
```

Comment:
The corner radius is set in `PrimaryRoundedButton` already, so we can remove it
```suggestion
```

same comment for `button.titleLabel?.font` and `contentPadding`. We'll probably be able to remove those UX constants as well.

Reply:
We should remove `button.titleLabel?.font` and `contentPadding` and clean up the UX constants. I didn't make a new comment for each properties.